ehr/src/org/labkey/ehr/pipeline/GeneticCalculationsImportTask.java

```diff
@@ -689,8 +689,8 @@ private static void processInbreeding(Container c, User u, File pipelineDir, Log
                     continue;
                 }
 
-                //row.put("Id", subjectId);
-                row.put("participantid", subjectId);
+                row.put("Id", subjectId);
+                //row.put("participantid", subjectId);
```

Comment:
@labkey-jeckels: isnt this commented out?

Reply:
Ha, yes. Sorry. Your comment pushed it below the visible section of the page when I was looking at the diff.